blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/HistogramSerializationTest.java

```diff
@@ -13,12 +13,10 @@
  *    See the License for the specific language governing permissions and
```

Comment:
FYI - @tilogaat says that the histogram API is not used by customers and will eventually be removed.  Not saying we shouldn't test it (perhaps this isn't even related to it) but good to be aware of when expanding the test coverage.